script/sync-ghes/index.ts

```diff
@@ -33,15 +43,18 @@ async function checkWorkflows(
     for (const e of dir) {
       if (e.isFile()) {
         const workflowFilePath = join(folder, e.name);
-        const enabled = await checkWorkflow(workflowFilePath, enabledActions);
-
         const workflowId = basename(e.name, extname(e.name));
-        const workflowProperties = require(join(
+        const workflowProperties: WorkflowProperties = require(join(
           folder,
           "properties",
           `${workflowId}.properties.json`
         ));
         const iconName: string | undefined = workflowProperties["iconName"];
+        const partnerWorkflow = workflowProperties.categories === null;
```

Comment:
Unfortunately right now this'll block the `blank` category - https://github.com/github/github/blob/master/config/actions_workflow_templates.json#L53 - would it be better as another configuration in your `settings.json` file do you think vs the null comparison? Or don't add the list but make an exception to the exception for `blank`?

Reply:
Ah good catch. I was trying to avoid having another configuration that we need to keep in sync. Let me think about it, maybe special casing `blank` is the easiest solution for now.. :/

Reply:
@cdb  k, updated